config/locales/es.yml

```diff
@@ -3594,8 +3594,9 @@ es:
           title: Puedes calificar para una Deducción o Crédito por impuestos sobre la propiedad en tu vivienda principal, ya sea que la hayas comprado o alquilado.
       nj_ineligible_property_tax:
         edit:
-          continue_text: Haz clic en continuar para seguir trabajando en tus impuestos.
+          continue_text: Haz clic en "Continuar" para seguir trabajando en tus impuestos.
           go_back_link: Corregir.
+          housing_assistance_html: (Si te interesa la asistencia de vivienda, llama al 211 o <a href="https://www.nj.gov/dca/dhcr/offices/housing_assistance.shtml" target="_blank" rel="noopener nofollow">visita la página del Departamento de Asuntos Comunitarios de New Jersey sobre Asistencia de Vivienda</a>.)
```

Comment:
[dust] Does a Spanish language version of this website exist?

Reply:
I don't believe so unfortunately :( Part of the value HomeX is providing